parsers/ENTE.py

```diff
@@ -25,6 +27,10 @@
 }
 
 
+def floor_to_minute(dt: datetime) -> datetime:
+    return dt.replace(second=0, microsecond=0)
+
+
 def fetch_production(
```

Comment:
I've noticed that when running `poetry run test_parser HN production` from the command line, the `fetch_production` function of the **HN** parser is called, not the ENTE parser. So I'm wondering if the `fetch_production` funtion of the ENTE parser is actually used or needed at all?

Reply:
Looks like ENTE is only used for exchanges at the moment so no it would not be needed. I would not recommend removing parser functions but since this only outputs unknown data I don't think we would use it anyhow as it is now. But if it's not causing any larger issues I'd say leave it.

Reply:
Thank you for the clarification. I will leave it as it is then, as this is not a big issue.